### ISSUE TITLE
switch to uuid v5 or even away from uuid

### ISSUE THREAD

**Issue by cmawhorter** [cmawhorter/eyed#6]
the goal of this lib should be to create a guaranteed collision-free id (as guaranteed as possible).  

currently, it uses v4 which could lead to problems when used with inconsistent databases.

explore uuid v5 or possibly abandoning uuid entirely.  it might always be nice to introduce a "strategy" option that allows the type of code generated be configured e.g. `strategy: lowvolume` => `id_short123` or `strategy: highvolume` => `id_verylongmaybelongerthan16bytestoguaranteenocollisions`.

**Comment by cmawhorter** [comment]
uuid 4 is essentially just random and good enough for most uses.  strategies seemed like overkill.  added ability to request fewer bytes to handle lower volume.  it's possible to use v5 uuid now by base62 encoding yourself.

tl;dr not going to do this and ended up going different route
